numpy/f2py/tests/test_compile_function.py

```diff
@@ -0,0 +1,121 @@
+"""See https://github.com/numpy/numpy/pull/11937.
```

Comment:
I'm not sure this reference is useful

Reply:
I sort of went back and forth, but on looking into what some of the other modules tested it simply seemed easier to link to the PR than trying to explain it ;) The whole series could use better documentation but it will be a dirty job.